### ISSUE TITLE
storybook/no-uninstalled-addons false positives on monorepo

### ISSUE THREAD

**Issue by unional** [storybookjs/eslint-plugin-storybook#127]
**Describe the bug**

Getting errors like these in a monorepo:

```sh
The @storybook/addon-links is not installed in pats/. Did you forget to install it or is your package.json in a different location?
```

the `pats` is the root of the monorepo.
Each package has storybook and addons installed.

**Expected behavior**
no error

**Additional context**
I'm not sure if storybook should be hoisted or install only at the top level of monorepo.

Right now, I have each package self-contained and each has their own copy of storybook.

I'm using `pnpm`.

**Comment by yannbf** [comment]
Hey @unional thanks for opening this issue. Would you be able to either provide a minimal reproduction repo or a PR to fix this issue? This is the rule code: https://github.com/storybookjs/eslint-plugin-storybook/blob/main/lib/rules/no-uninstalled-addons.ts

You could for now disable that rule to unblock you.

**Comment by barrymichaeldoyle** [comment]
@yannbf this isn't exactly a fix but it looks like a work around has been implemented here that is better than just blanket ignoring the rule:

https://github.com/storybookjs/eslint-plugin-storybook/pull/129

**Comment by clarkd** [comment]
We have the same issue here - also using pnpm with a monorepo.
